dispersion_protocol/messages.md

```diff
@@ -0,0 +1,183 @@
+| uid                 | `uint64_t` |                      | Invalid:0                                                   | Integer to uniquely identify this hardware. (0 if unknown)                                                       |
+| cap_flags           | `uint16_t` |                      | [DISPERSION_DEVICE_CAP_FLAGS](#dispersion_device_cap_flags) | Bitmap of dispersion device capability flags.                                                                    |
+| custom_cap_flags    | `uint16_t` |                      |                                                             | Bitmap for use for dispersion-specific capability flags. flags.                                                  |
+| dispersion_rate_min | `float`    | liters/min or kg/min | invalid:NaN, <0                                             | Minimum dispersion rate this device can support                                                                  |
+| dispersion_rate_max | `float`    | liters/min or kg/min | invalid:NaN, <=0                                            | Maximum dispersion rate this device can support                                                                  |
+| pressure_min        | `uint32_t` |          Pa          |                                                             | If this is a spray dispersion system, minimum pressure the hardware supports independent of nozzle configuration |
+| pressure_max        | `uint32_t` |          Pa          | invalid:0                                                   | If this is a spray dispersion system, maximum pressure the hardware supports independent of nozzle configuration |
+
+## DISPERSION_DEVICE_STATUS
+
+Message reporting the low level status of a dispersion device.
+
+This message should be published a low regular rate (e.g. 5 Hz) but also during key events such a flag change. It should also be published in response to a request.
+
+| Field Name        | Type       |        Units         | Values                                                            | Description                                                                    |
+| :---------------- | :--------- | :------------------: | :---------------------------------------------------------------- | :----------------------------------------------------------------------------- |
+| target_system     | `uint8_t`  |                      |                                                                   | System ID uniquely identifying the vehicle this payload device is attached to  |
+| target_component  | `uint8_t`  |                      |                                                                   | Component ID uniquely identifying this device on the system                    |
```

Comment:
YOu don't need these two. Target ids are to direct a message to a particular recipient - the source (which is what your description indicates) is in every message in the header.

```suggestion
```